city_scrapers/spiders/chi_standards_tests.py

```diff
@@ -0,0 +1,82 @@
+import datetime
+
+from city_scrapers_core.constants import COMMITTEE
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiStandardsTestsSpider(CityScrapersSpider):
+    name = "chi_standards_tests"
+    agency = "Chicago Committee on Standards and Tests"
+    timezone = "America/Chicago"
+    start_urls = [
+        "https://www.chicago.gov/city/en/depts/bldgs/supp_info/committee_on_standardsandtests.html"
+    ]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        for item in response.xpath("//div[@class='col-xs-12']/table[1]//td/p"):
+            if self._pass_filter(item) is False:
+                continue
+            meeting = Meeting(
+                title="Committee on Standards and Tests",
+                description="",
+                classification=COMMITTEE,
+                start=self._parse_start(item),
+                end=None,
+                all_day=False,
+                time_notes="Confirm details with the agency",
+                location={
+                    "address": "121 North LaSalle Street, Room 906, Chicago, Illinois 60602",
```

Comment:
Small thing, but we've been using the state abbreviation `IL` in places where we're statically setting the address. It might also be worth validating that it occurs in room 906 with a `_validate_location` function https://cityscrapers.org/docs/development/#location

Reply:
Understandable, but the website does not include an address for the meetings. The statically set address here was deduced from the the few post-meeting pdfs provided on the website. I scanned through all the provided pdfs and they all had the same address, forming the basis of my assumption that the meetings are all held at that location.

As far as I can tell, the website provides insufficient information to parse or validate the meeting locations ahead of time. 

If there is something that I have missed in parsing/validating meeting locations, please do let me know. Otherwise, I don't know how else I can parse the location and will proceed with updating the state abbreviation.

Reply:
Makes sense, in that case just updating the abbreviation is fine